src/templates/index.pug

```diff
@@ -50,7 +67,7 @@ body.layout
 
     // Sidebar
     div.sidebar#sidebar.hidden
-        +sidebar_link("Introduction", "Introduction")
-        +sidebar_link("0", "Lesson 0: Getting Started")
+        each lesson in lessons
+            +sidebar_link(lesson.id, `${lesson.num}: ${lesson.name}`)
```

Comment:
We should make sure to properly format `${lesson.num}`, especially given that it's handled as a float. I would like to not have `1.299999` as a lesson by accident, due to some rounding errors (1.3 can not be represented as a float, the closest are `1.2999999523162841796875`  or `1.30000007152557373046875`).